.github/workflows/publish.yml

```diff
@@ -36,10 +36,10 @@ jobs:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: publish coverage
-        uses: coverallsapp/github-action@v1.0.1
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-        env:
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+      # - name: publish coverage
+      #   uses: coverallsapp/github-action@v1.0.1
+      #   with:
+      #     github-token: ${{ secrets.GITHUB_TOKEN }}
+      #   env:
+      #     COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
```

Comment:
![michael scott upset](https://user-images.githubusercontent.com/10633734/109677745-3f1ca300-7b7a-11eb-8b56-601ba17fcc43.jpg)

Reply:
The test result folders are deleted before this task runs, I just commented this out temporarily and will uncomment it after we decide will to put the test result folder

Reply:
I will update the coverage path. Sorry for commented it out